<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>551</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>4A8G0039_OpenCV</string>
  </property>
  <property name="iconSize">
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QLabel" name="Img_Lable">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="tabletTracking">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="Top_menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="File_menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="OpenFile_action"/>
    <addaction name="separator"/>
    <addaction name="SaveFile_action"/>
   </widget>
   <widget class="QMenu" name="Setting_menu">
    <property name="title">
     <string>Setting</string>
    </property>
    <addaction name="ROI_action"/>
    <addaction name="separator"/>
    <addaction name="Show_histogram_action"/>
    <addaction name="separator"/>
    <addaction name="Show_change_colorspace_action"/>
   </widget>
   <widget class="QMenu" name="menuImage_Processing">
    <property name="title">
     <string>Image Processing</string>
    </property>
    <addaction name="action"/>
    <addaction name="separator"/>
    <addaction name="action_3"/>
   </widget>
   <addaction name="File_menu"/>
   <addaction name="Setting_menu"/>
   <addaction name="menuImage_Processing"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="OpenFile_action">
   <property name="text">
    <string>OpenFile</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="ROI_action">
   <property name="text">
    <string>設定ROI</string>
   </property>
  </action>
  <action name="Show_histogram_action">
   <property name="text">
    <string>顯示影像直方圖</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>影像二值化</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>直方圖等化</string>
   </property>
  </action>
  <action name="Show_change_colorspace_action">
   <property name="text">
    <string>顯示或改變色彩空間</string>
   </property>
  </action>
  <action name="SaveFile_action">
   <property name="text">
    <string>SaveFile</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
